src/ElfUtils/ElfFile.cpp

```diff
@@ -385,4 +388,11 @@ ErrorMessageOr<std::unique_ptr<ElfFile>> ElfFile::Create(
   return result;
 }
 
+ErrorMessageOr<uint32_t> ElfFile::CalculateDebuglinkChecksum(
+    const std::filesystem::path& file_path) {
+  ErrorMessageOr<std::string> buffer_or_error = orbit_base::ReadFileToString(file_path);
```

Comment:
as far as I understand this file could be quite big (it most likely has .debug_info section)... maybe we will need to change to use iterative crc32 calculation instead?

Reply:
(add a todo?)

Reply:
Yes, very good point. As discussed OOB, I will add a TODO and change that to an interative approach when `orbit_base::ReadFully` is implemented in a couple of days.

Reply:
Added the TODO

Reply:
Since `ReadFully` is already implemented, I will wait until it's merged.

Reply:
@dimitry- I changed the implementation. PTAL!